ProteoformSuiteInternal/TopDownProteoform.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Proteomics;
+
+namespace ProteoformSuiteInternal
+{
+    public class TopDownProteoform : Proteoform
```

Comment:
Yes when I first added TopDownProteoform I went back and forth between inheriting from Experimental and Theoretical because there is a lot of overlap from both. I do like that concept of 2 ExperimentalProteoforms... might be nice for my work. There's many more ExperimentalIntactProteoforms than ExperimentalTopDownProteoforms.